flow.js

```diff
@@ -0,0 +1,20 @@
+export const flowRules = {
+  'flowtype-errors/show-errors': 2,
+  'flowtype/generic-spacing': 0,
+  'flowtype/no-types-missing-file-annotation': 2,
+  'flowtype/no-weak-types': 2,
+  'flowtype/require-parameter-type': [
+    2,
+    { excludeArrowFunctions: 'expressionsOnly' },
```

Comment:
i added this

Reply:
I kinda disagree with this addition. What's your reasoning?

Reply:
it pairs with the `require-return-type` rule so that you don't have to do `(x: string) => x`. you can just do `(x) => x`

Reply:
Right, but what's the reasoning behind making that looser? In general we keep our Flow rules pretty strict.

Reply:
im hoping it will make turning this rule on in seller less painful. there are 775 errors with it just set to `2` 😭 vs 208 with it on

Reply:
I would override that in that app's Eslintrc, then.